docs/examples/fork-process-model/README.rst

```diff
@@ -1,7 +1,7 @@
 Working With Fork Process Models
 ================================
 
-The `BatchSpanProcessor` is not fork-safe and doesn't work well with application servers
+`In python versions 3.6 and older <https://github.com/open-telemetry/opentelemetry-python/pull/2242>`_, the `BatchSpanProcessor` is not fork-safe and doesn't work well with application servers
```

Comment:
What is the purpose of this change? OpenTelemetry doesn't support Python versions that are less than 3.6 so why call this out specifically?